src/osemgrep/cli_scan/Rule_filtering.ml

```diff
@@ -15,12 +17,10 @@
 (*****************************************************************************)
 (* Types *)
 (*****************************************************************************)
-type product = SAST | SCA | Secrets | Interfile [@@deriving show]
-
 type conf = {
   exclude_rule_ids : Rule_ID.t list;
   severity : Rule.severity list;
-  exclude_products : product list;
+  exclude_products : Out.product list;
```

Comment:
supportive of removing this 'type product' and reuse Out.product, but not that Interfile
is not in product. Just FYI.

Reply:
Yea, I agree it's cleaner